docs/src/user_docs/assembly/cryptographic_operations.md

```diff
@@ -4,7 +4,7 @@ Miden assembly provides a set of instructions for performing common cryptographi
 ### Hashing and Merkle trees
 [Rescue Prime Optimized](https://eprint.iacr.org/2022/1577) is the native hash function of Miden VM. The parameters of the hash function were chosen to provide 128-bit security level against preimage and collision attacks. The function operates over a state of 12 field elements, and requires 7 rounds for a single permutation. However, due to its special status within the VM, computing Rescue Prime Optimized hashes can be done very efficiently. For example, applying a permutation of the hash function can be done in a single VM cycle.
 
-| Instruction                      | Stack_input        | Stack_output      | Notes                                                                                                                                                                                                                                                                                                                                                  |
+| Instruction                      | Stack input        | Stack output      | Notes                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
The main reason for using underscore between the words is that otherwise, these columns get shrunk and the table becomes much less readable. I agree that we should standardize but I wonder if it still makes sense to keep the underscore and just use `Stack_inputs` and `Stack_outputs` everywhere.

Reply:
I will change everything to `Stack_input` and `Stack_output`